api/processor_api.py

```diff
@@ -97,8 +104,6 @@ async def validation_exception_handler(request: Request, exc: RequestValidationE
             app.add_middleware(CORSMiddleware, allow_origins="*", allow_credentials=True, allow_methods=["*"],
                                allow_headers=["*"])
 
-        app.mount("/static", StaticFiles(directory="/repo/data/processor/static"), name="static")
```

Comment:
Without this, can the frontend still access the files in this directory? I don't think so. For example the video files. The csv files are not used directly anymore?

Reply:
I changed the `StaticFiles` into a router to include the authentication middleware (api/routers/static.py).
After some research, I did;t find any way to force authentication using `StaticFiles `.